src/LabelSelector/LabelChip/LabelChip.stories.tsx

```diff
@@ -79,3 +79,21 @@ export const Small = {
 
   render: Template
 };
+
+export const Selected = {
```

Comment:
I expected this story to allow me to click on the LabelChip to select/deselect it. I think this is possible with the `onClick` property but it would be good to demonstrate this use case as per the Figma.

Reply:
Story updated. Indroduced a new `showIcon` prop to decide to show the icon.